### PR TITLE
Update map template url parameters

### PR DESCRIPTION
**Pull request by andreeaceachir142**
**Note**
Setting this as a draft pull request until getting the feature released

### CODE REVIEW

File: docs/10-Map Templates/web-map-template.mdx [hunk 1]
```diff
@@ -49,6 +49,9 @@ To have any Stencil component changes reflected in this project, you must run `n
 
 ### Supported properties on the MapsIndoors Map component​
 
+The MapsIndoors Map component supports a list of props (see the list below) that can be configured in the `App.jsx` file. 
+When loading the MapsIndoors Map component for the first time, the map will respect the default values set for the `apiKey`, `venue`, `logo` and `primaryColor` props, which can be found in the `MapsIndoorsMap.jsx` file under the `defaultProps` object.
```
Comment (markhougaard) [inline docs/10-Map Templates/web-map-template.mdx:53]: In Markdown, you have to have an empty line between two paragraphs, or the linter will complain about it not being valid.

```suggestion
The MapsIndoors Map component supports a list of props (see the list below) that can be configured in the `App.jsx` file.

When loading the MapsIndoors Map component for the first time, the map will respect the default values set for the `apiKey`, `venue`, `logo` and `primaryColor` props, which can be found in the `MapsIndoorsMap.jsx` file under the `defaultProps` object. 
```

File: docs/10-Map Templates/web-map-template.mdx [hunk 2]
```diff
@@ -122,6 +126,20 @@ Use the web component in your HTML:
 
 Add attributes to the web component as needed (see supported properties below).
 
+**Note!** The `external-IDs` and `app-user-roles` expect an array, which in a web component is handled differently (see example below).
+In your script, define the array of external IDs that you want to be shown on the Map Template. 
+Then get a hold of the mapsIndoors map element using the `document.querySelector()` method. 
+When you have the mapsIndoors map element, assign its prop `externalIDs` the array of external IDs that you defined at the beginning.
```
Comment (markhougaard) [inline docs/10-Map Templates/web-map-template.mdx:132]:
```suggestion
**Note!** The `external-IDs` and `app-user-roles` expect an array, which in a web component is handled differently (see example below).

In your script, define the array of external IDs that you want to be shown on the Map Template. Then get a hold of the `mapsIndoorsMapElement` using the `document.querySelector()` method. When you have the `mapsIndoorsMapElement`, assign its prop `externalIDs` the array of External IDs that you defined at the beginning.
```

File: docs/10-Map Templates/web-map-template.mdx [hunk 3]
```diff
@@ -122,6 +126,20 @@ Use the web component in your HTML:
...
+```javascript
+const externalIDsArray = ["externalID-1", "externalID-2"]
+const mapsIndoorsMapElement = document.querySelector('mapsindoors-map')
+mapsIndoorsMapElement.externalIDs = externalIDsArray;
+```
+
+Use query parameters to configure the web component by setting the `supports-url-parameter` attribute to `true`.
+
+
 ### Using just the browser
```
Comment (markhougaard) [inline docs/10-Map Templates/web-map-template.mdx:143]:
```suggestion
Use query parameters to configure the web component by setting the `supports-url-parameter` attribute to `true`.

### Using just the browser
```

File: docs/10-Map Templates/web-map-template.mdx [hunk 4]
```diff
@@ -155,6 +173,7 @@ Add attributes to the web component as needed (see supported properties below).
 ```
 
 Add attributes to the web component as needed (see supported properties above).
+Use query parameters to configure the web component by setting the `supports-url-parameter` attribute to `true`.
```
Comment (markhougaard) [inline docs/10-Map Templates/web-map-template.mdx:176]:
```suggestion
Add attributes to the web component as needed (see supported properties above).

Use query parameters to configure the web component by setting the `supports-url-parameter` attribute to `true`.
```

File: docs/10-Map Templates/web-map-template.mdx [hunk 5]
```diff
@@ -186,10 +205,12 @@ import MapsIndoorsMap from '@mapsindoors/map-template/dist/mapsindoors-react.es.
 ```
 
 Add properties to the MapsIndoorsMap component as needed (see list above).
+Use query parameters to configure the MapsIndoorsMap component by setting the `supportsUrlParameters` prop to `true`.
```
Comment (markhougaard) [inline docs/10-Map Templates/web-map-template.mdx:208]:
```suggestion
Add properties to the MapsIndoorsMap component as needed (see list above).

Use query parameters to configure the MapsIndoorsMap component by setting the `supportsUrlParameters` prop to `true`.
```

File: docs/10-Map Templates/web-map-template.mdx [hunk 6]
```diff
@@ -186,10 +205,12 @@ import MapsIndoorsMap from '@mapsindoors/map-template/dist/mapsindoors-react.es.
...
 
 ## Query Parameters
 
-The Map Template supports using query parameters for all the properties provided by the MapsIndoorsMap component.
+The Map Template supports using query parameters for all the properties provided by the MapsIndoorsMap component if the `supportsUrlParameters` property is set to true.
+In addition, the Web component and the React component also support using the URL parameters if the `supports-url-parameters` attribute respectively the `supportsUrlParameters` prop is set to `true`.
```
Comment (markhougaard) [inline docs/10-Map Templates/web-map-template.mdx:213]:
```suggestion
The Map Template supports using query parameters for all the properties provided by the MapsIndoorsMap component if the `supportsUrlParameters` property is set to true.

In addition, the Web component and the React component also support using the URL parameters if the `supports-url-parameters` attribute or the `supportsUrlParameters` prop is set to `true`.
```

**Review by markhougaard — CHANGES_REQUESTED** [review]
@andreeaceachir142 A general note: Should we write "Web Component" or "web component"? I can go with either, but we should just be consistent.

**Comment by andreeaceachir142** [comment]
Let's go with Web Component and React Component, I have seen them being used like this more often. @markhougaard 
I'll make sure it's changed everywhere.

**Review by markhougaard — APPROVED** [review]

**Pull request merged**